index.html

```diff
@@ -2462,7 +2462,7 @@ <h5>Existing implementations</h5>
 </dd>
 <dt>google-maps-embed</dt>
 <dd><a>supported, with limitations</a>:
-  You are able to generate a link to the map that can be shared via a variety of methods, but there is no way way to easily this information to another application.
+  You are able to generate a link to the map that can be shared via a variety of methods, but there is no way to easily send this information to another application.
```

Comment:
> but there is no way to easily send this information to another application.

is "export" preferred over "send" perhaps?

Reply:
"send" works. Thanks for the catch & sorry I didn't notice this PR sooner.